Assets/_BForBoss/_Weapons/Scripts/Weapons/Weapon.cs

```diff
@@ -2,6 +2,7 @@
 using FMODUnity;
 using PerigonGames;
 using UnityEngine;
+using UnityEngine.VFX;
```

Comment:
I don't think this is needed here?

Reply:
You're right, forgot to remove it from when I was testing where to put the VFX code.